mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -353,6 +353,8 @@ public void onSharedPreferenceChanged(final SharedPreferences sharedPreferences,
         } else if (key.equals("pref_popup")) {
             final boolean enablePopup = mSharedPrefs.getBoolean("pref_popup", false);
             mIitcWebView.setSupportPopup(enablePopup);
+        } else if (key.equals("pref_webview_zoom")) {
+            mIitcWebView.setWebViewZoom(mSharedPrefs.getInt("pref_webview_zoom", -1));
```

Comment:
```suggestion
            mIitcWebView.setWebViewZoom(Integer.parseInt(mSharedPrefs.getString("pref_webview_zoom", "-1")));
```

Reply:
it sounds like the pref is not stored as a int

Reply:
confirmed, it does not look possible to store an integer using `ListPreference`
my bad (since I suggested the reversed change...)

Reply:
Why does it work then? 🤔